### ISSUE TITLE
Filter /trends by cohorts

### ISSUE THREAD

**Issue by jamesefhawkins** [PostHog/posthog#169]
## New spec by @timgl 

We need to be able to select multiple cohorts under breakdown in /trends. Once selected, it should split out each of the entities in that trends view by those cohorts.

For example:
2 cohorts:
- user property premium_user = true
- user property premium_user = false

I want to be able to see the volume of a certain event for both of those cohorts.

## Previously...
**Is your feature request related to a problem? Please describe.**
I can't easily see which users (or groups/teams) are really engaged, and which aren't. 

User cohorts would help me understand the kinds of people and companies that love our stuff, and equally, it'd help me work out who doesn't love it.

If we did that, I'd know who to speak to, to build a better product.

This quickly will make me want to answer a new question - "where are my best users coming from?"

If we did that, I could grow my product more easily, and have more conversations with users, to build a better product!

In PostHog's case, this would be users who actually made an account. I want to see that list of users, next to their UTM tags so I'd know which marketing channel they came from.

## Creating cohorts

To do this, I need to be able to create a group of users based on (i) their event history (INCLUDING within a certain timeframe ie users who did X within the last Y days" or (ii) their properties. That lets me quickly see my "best" users or my "worst" ones.

**Importantly**, this would be the first step in enabling team-level analytics.

With that group, I then need to be able to draw a table similar to the funnel breakout - I need to see every user listed, next to a column containing a property that i select.

On the /actions/trends page, we then need to be able to filter by cohort, to show usage information for just that group. This would mean PostHog users could understand how much individual teams are using the tool, or what the user behaviour looks like for engaged users versus non engaged.

I need this group to persist so I don't have to reconfigure it several times.

## Tracking groups over time

Being able to produce a graph like the /trends/ for these groups (the volume in each), would mean it's possible to understand if they are varying over time (do we have MORE engaged users or fewer)

**Describe alternatives you've considered**
This would require complex SQL.

**Comment by Tannergoods** [comment]
Requested by: 
https://app.hubspot.com/contacts/6958578/contact/30355
https://app.hubspot.com/contacts/6958578/contact/28851/

**Comment by Tannergoods** [comment]
We have built cohorts already but this has not closed as far as I can see due to the lack of graphical interface? 

This is also part of Parity project.

Essentially we should be able to apply Trends graphs to cohorts we have already defined. If we apply the same logic we would also need to visualize the graphs as a % of the entire cohort - not just Total or DAUs.

A key part of this issue is also the ability to select multiple cohorts within a graphical interface so as to compare them

Cohorts could also be viewed as Paths
